backend/src/Helper.js

```diff
@@ -1,23 +1,33 @@
+const movieFilter = ["id", "title", "overview", "poster_path", "release_date"]
+const watchLaterFilter = ["id", "watched"]
```

Comment:
move each filter to its approriate function, the movieFilter have it inside the filterMovie function, they dont need to be global.